src/Groups/directproducts.jl

```diff
@@ -71,33 +74,44 @@ The parameter `morphisms` is `false` by default. If it is set `true`, then
 the output is a triple (`G`, `emb`, `proj`), where `emb` and `proj` are the
 vectors of the embeddings (resp. projections) of the direct product `G`.
 """
-function inner_direct_product(L::AbstractVector{T}; morphisms=false) where T<:Union{PcGroup,FPGroup}
-   P = GAP.Globals.DirectProduct(GapObj([G.X for G in L]))
-   DP = T(P)
-   if morphisms
-      emb = [GAPGroupHomomorphism(L[i],DP,GAP.Globals.Embedding(P,i)) for i in 1:length(L)]
-      proj = [GAPGroupHomomorphism(DP,L[i],GAP.Globals.Projection(P,i)) for i in 1:length(L)]
-      return DP, emb, proj
-   else
-      return DP
-   end
+function inner_direct_product(
+  L::AbstractVector{T}; morphisms=false
```

Comment:
```suggestion
  L::AbstractVector{T}; morphisms::Bool=false
```
Maybe also adjust the docstring to refer to `morphisms` as "keyword argument" instead of "parameter" ?

Reply:
Done in a65db37ee2740413118425c901feb2c665d30035